doc/source/rest.j2

```diff
@@ -39,6 +39,11 @@ To create a |metric|, the following API request should be used:
 
 {{ scenarios['create-metric']['doc'] }}
 
+.. note::
+
+  Once the |metric| is created, it cannot be PATCHed to modify the attributes.
```

Comment:
You can modify the resource and name, but not by patching but by going through the resource API.

I think just saying that you cannot change the archive policy is what we want here.

Reply:
Thanks, there are so many details I have to pick up like this. I really think the document should be upgraded to improve the learning curve. Most information is there but not organized well.

Reply:
I just digged into the document and was not able to find where it mentioned metric's name and resource modification.